launch_ros/launch_ros/actions/load_composable_nodes.py

```diff
@@ -82,6 +86,21 @@ def __init__(
         self.__final_target_container_name = None  # type: Optional[Text]
         self.__logger = launch.logging.get_logger(__name__)
 
+    @classmethod
+    def parse(cls, entity: Entity, parser: Parser):
+        """Parse load_composable_node."""
+        _, kwargs = super().parse(entity, parser)
+
+        kwargs['target_container'] = parser.parse_substitution(
+            entity.get_attr('target', data_type=str))
+
+        composable_nodes = entity.get_attr('composable_node', data_type=List[Entity])
+        if composable_nodes is not None:
+            parsed_tuples = [ComposableNode.parse(parser, entity) for entity in composable_nodes]
+            kwargs['composable_node_descriptions'] = [t[0](**t[1]) for t in parsed_tuples]
+
```

Comment:
you also need here a

```python3
for composable_node in composable_nodes:
  composable_node.assert_entity_completely_parsed()
```

Reply:
Added in b1412a0.

Reply:
Moved to at the end of `ComposableNode.parse` in da1f68c.